### PR TITLE
solutions pillar review

### PR DESCRIPTION
**Pull request by jacobdadams**
Review of the by-user side of the solutions pillar

### CODE REVIEW

File: src/pages/solutions/for-gis-users.astro [hunk 1]
```diff
@@ -25,11 +25,11 @@ const page: IStandardPageMetadata = {
...
   section: stripeSections([
     {},
-    { title: 'The SGID', actionUrl: '/products/sgid' },
-    { title: 'UGRC API', actionUrl: '/products/api' },
-    { title: 'Address geocoding', actionUrl: '/products/sgid/address/api-client' },
-    { title: 'Base maps and imagery', actionUrl: '/products/discover' },
-    { title: 'GPS location services', actionUrl: '/products/turn' },
+    { title: 'The SGID', actionUrl: '/products/sgid', actionText: 'Explore the SGID' },
+    { title: 'UGRC API', actionUrl: '/products/api', actionText: 'Get connected' },
+    { title: 'Address geocoding', actionUrl: '/products/sgid/address/api-client', actionText: `Get geocodin'` },
```
Comment (steveoh) [inline src/pages/solutions/for-gis-users.astro:30]:
```suggestion
    { title: 'Address geocoding', actionUrl: '/products/sgid/address/api-client', actionText: `Start geocoding` },
```

File: src/pages/solutions/for-gis-users.astro [hunk 2]
```diff
@@ -46,26 +46,26 @@ const page: IStandardPageMetadata = {
...
-      SGID. We make it easy by making a wide array of data available in one searchable location. The SGID Index is a
-      great place to start your exploration. The search results will point you to more information about the data, its
-      update schedule, and where and how to access it. SGID data is available in various formats, including web map
-      services and direct database access. Visit the <a href="/products/sgid">SGID landing page</a> to learn more.
+      The SGID is our premier curated library of Utah-related geospatial data - the perfect starting point for your map
+      data needs. We work with state and local governments and federal partners to aggregate and share a wide variety of
+      authoritative data in one searchable location. The <a href="/products/sgid/sgid-index">SGID Index</a> is a great place
+      to start your exploration. The search results will point you to more information about the data, its update schedule,
+      and where and how to access it. SGID data is available in various formats, including web map services and direct database
+      access. Visit the <a href="/products/sgid">SGID landing page</a> to learn more.
```
Comment (steveoh) [inline src/pages/solutions/for-gis-users.astro:68]:
```suggestion
      access. Visit the <a href="/products/sgid">SGID product page</a> to learn more.
```

File: src/pages/solutions/for-gis-users.astro [hunk 3]
```diff
@@ -92,26 +92,32 @@ const page: IStandardPageMetadata = {
...
       be mapped! Our address locator is one of the most popular products we offer. Depending on your needs, there are a
       variety of ways to interact with it including through the UGRC API, an installable desktop application (macOS and
-      Windows), or through the locator tab in ArcGIS Pro. Our locator accepts a variety of input values including civic
-      addresses and latitude and longitude coordinates. Depending on the locator style you choose, users can pass in a
-      single address or a list of addresses. You can learn more, including how to access them, on our address locator
-      page.
+      Windows), or through the locator tab in ArcGIS Pro. Our locator accepts a variety of input values including common
+      addresses or latitude and longitude coordinates. Depending on the locator style you choose, users can pass in a
+      single address or a list of addresses. You can learn more, including how to access our locators, on our address
+      locator page.
```
Comment (steveoh) [inline src/pages/solutions/for-gis-users.astro:98]: do we have a locator page?

**Review by steveoh — APPROVED** [review]

File: src/pages/solutions/for-gis-users.astro (hunk 3 above)
Comment (jacobdadams) [reply src/pages/solutions/for-gis-users.astro:98]: good call. fixed to refer to geocoders and the address data pages.

**Review by steveoh — APPROVED** [review]

**Pull request merged**